test/lit.cfg

```diff
@@ -982,7 +982,8 @@ elif run_os == 'linux-androideabi':
                   'prebuilt', 'linux-x86_64', 'arm-linux-androideabi', 'bin'),
         make_path(test_resource_dir, config.target_sdk_name))
     # The Swift interpreter is not available when targeting Android.
-    config.available_features.remove('swift_interpreter')
+    if 'swift_interpreter' in config.available_features:
+        config.available_features.remove('swift_interpreter')
```

Comment:
Okay, note for those browsing this: this is in the android-specific path, and removing a non-existent value is an error in python *sigh*